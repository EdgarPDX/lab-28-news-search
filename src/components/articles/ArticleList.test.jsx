import React from 'react';
import { render, cleanup } from '@testing-library/react'
import ArticleList from './ArticleList';

describe('Article component', () => {
    afterEach(() => cleanup());
    it('renders Article', () => {
        const { asFragment } = render(<ArticleList
            articles={[
                { title:"Test Title", author:"Test Author", description:"Test Description" }
            ]}
            />);
        expect(asFragment()).toMatchSnapshot()
    });
});