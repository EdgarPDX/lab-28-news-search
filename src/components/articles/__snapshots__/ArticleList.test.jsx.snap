// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component renders Article 1`] = `
<DocumentFragment>
  <ul
    class="ArticleList"
  >
    <li>
      <div
        class="Article"
      >
        <h3>
          Test Title
        </h3>
        <h4>
          Test Author
        </h4>
        <p>
          Test Description
        </p>
      </div>
    </li>
  </ul>
</DocumentFragment>
`;
